modules/dag-shared/src/main/scala/org/tessellation/dag/dag.scala

```diff
@@ -0,0 +1,15 @@
+package org.tessellation
+
+import org.tessellation.dag.domain.block.{BlockReference, DAGBlock}
+import org.tessellation.dag.snapshot.{GlobalSnapshot, SnapshotOrdinal, StateChannelSnapshotWrapper}
+
+package object dag {
+
+  val dagKryoRegistrar: Map[Class[_], Int] = Map(
+    classOf[GlobalSnapshot] -> 400,
+    classOf[StateChannelSnapshotWrapper] -> 401,
+    classOf[SnapshotOrdinal] -> 402,
+    classOf[DAGBlock] -> 403,
+    classOf[BlockReference] -> 404
```

Comment:
These are under different numbers in `dag-L1` module which may possibly lead to errors.

Reply:
I didn't realize that! I think we should remove it then from `dag-l1` and keep it only in `dag-shared`.

Reply:
yup, things should be defined only in its own registrar. we should avoid cross-module registration